L1Trigger/L1TMuonEndCap/src/bdt/Tree.cc

```diff
@@ -477,9 +479,15 @@ void Tree::loadFromXMLRecursive(TXMLEngine* xml, XMLNodePointer_t xnode, Node* t
   XMLAttrPointer_t attr = xml->GetFirstAttr(xnode);
   std::vector<std::string> splitInfo(3);
   if (xmlVersion >= 2017) {
-    for (unsigned int i = 0, j = 0; i < 10; i++) {
-      if (i == 3 || i == 4 || i == 6) {
-        splitInfo[j++] = xml->GetAttrValue(attr);
+    for (unsigned int i = 0; i < 10; i++) {
+      if (std::string("IVar") == xml->GetAttrName(attr)) {
+        splitInfo[0] = xml->GetAttrValue(attr);
+      }
+      if (std::string("Cut") == xml->GetAttrName(attr)) {
```

Comment:
An `else if` would have been more appropriate... but ok, this is not to be used for central productions